src/main/java/uk/gov/hmcts/divorce/document/content/NoticeOfProceedingContent.java

```diff
@@ -123,8 +123,8 @@ public class NoticeOfProceedingContent {
     private static final int PAPER_SERVE_OFFSET_DAYS = 28;
     private static final int RESPONDENT_SOLICITOR_RESPONSE_OFFSET_DAYS = 16;
 
-    private static final String APPLICANT_2_SIGN_IN_DIVORCE_URL = "applicant2SignInDivorceUrl";
-    private static final String APPLICANT_2_SIGN_IN_DISSOLUTION_URL = "applicant2SignInDissolutionUrl";
+    private static final String APPLICANT_2_SIGN_IN_DIVORCE_URL = "respondentSignInDivorceUrl";
+    private static final String APPLICANT_2_SIGN_IN_DISSOLUTION_URL = "respondentSignInDissolutionUrl";
```

Comment:
should we also rename constants to say respondent instead of applicant 2 ?

Reply:
Yeah, good point 👍